lib/rules/require-explicit-slots.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * @author Mussin Benarbia
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+
+/**
+ * @typedef {import('@typescript-eslint/types').TSESTree.TypeNode} TypeNode
+ */
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require slots to be explicitly defined with defineSlots',
+      categories: ['vue3-strongly-recommended'],
```

Comment:
I don't think it is necessary to change the configuration, so could you please change it to undefined?

```suggestion
      categories: undefined,
```